be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -213,7 +213,20 @@ void HdfsScannerContext::set_columns_from_file(const std::unordered_set<std::str
     }
 }
 
-void HdfsScannerContext::append_not_exised_columns_to_chunk(vectorized::ChunkPtr* chunk, size_t row_count) {
+void HdfsScannerContext::update_not_existed_columns_to_chunk(vectorized::ChunkPtr* chunk, size_t row_count) {
+    if (not_existed_slots.size() == 0) return;
+
+    ChunkPtr& ck = (*chunk);
+    ck->set_num_rows(row_count);
+    for (auto* slot_desc : not_existed_slots) {
+        auto col = ck->get_column_by_slot_id(slot_desc->id());
+        if (row_count > 0) {
+            col->append_default(row_count);
+        }
+    }
+}
+
+void HdfsScannerContext::append_not_existed_columns_to_chunk(vectorized::ChunkPtr* chunk, size_t row_count) {
```

Comment:
`update` ?

Reply:
no, this is another method, the update one is here: https://github.com/StarRocks/starrocks/pull/7413/files#diff-d87d2d93fc96f35903f3d611d309da8f4c2a675042ab62cfce0cba52e489e58fR216

Reply:
We still need `append_not_existed_columns_to_chunk` this method?